### ISSUE TITLE
Add "contructionYear" in real_estates

### ISSUE THREAD

**Issue by Sigri44** [lasconic/finary_uapi#173]
Add "contructionYear" into real_estates, to be compatible with RealT constructionYear Data

**Comment by lasconic** [comment]
"contructionYear" is available in finary api ?

**Comment by Sigri44** [comment]
Hum i don't know, but available on portfolio :
<img width="591" alt="image" src="https://github.com/lasconic/finary_uapi/assets/9304947/61933ce7-4d04-4828-9749-8ab9fccd65d9">

**Comment by nmathey** [comment]
`"building_period": "between_1975_1984",` ?

**Comment by lasconic** [comment]
Yes, we need a list of the possible value, and decide how we pass it to the function...
